code/datums/flock/flock.dm

```diff
@@ -228,6 +228,7 @@
 	if(!T)
 		return
 	src.traces -= T
+	hideAnnotations(T)
```

Comment:
What's reason for putting on removeTrace instead of logout?

Reply:
By the time `Logout` is called the client has already been transferred out of the mob, and you need the client to remove the image. I'm pretty sure the calls to `hideAnnotations` in logout never worked.

Reply:
Idk, I think this could still be put in logout? I think intention was just to remove client images while there's no client there, as showAnnotations() and hideAnnotations() is also done for flockdrones.